inscriptions/ISic003689.xml

```diff
@@ -198,76 +203,30 @@
-                <ab>
-                    <lb n="1"/>
-
-
-                     ἐνθάδε κῖτε Μαρκιλίνη,<lb n="2"/> 𐆠 σεμνὴ <unclear>κ</unclear>(αὶ)
-                    
-                    ἄμεμπτος
-                    <expan>
-                        <abbr>βιώσα</abbr>
-                        <ex>σα</ex>
-                    </expan>
-                    <lb n="3"/> ἰς τὸν κόσμον τοῦτον. ἀπε<supplied reason="omitted">κ</supplied>-<lb n="4"/>
-                     χώρι πρὸς <supplied reason="omitted">Κ</supplied>ύριον ἐτῶν<lb n="5"/> ιζʹ ΑΙΧΗΝ
-                    πράττουσα τῇ<lb n="6"/>α𐆠ω πρὸ θ
-                    <expan>
-                        <abbr>καλ</abbr>
-                        <ex>ανδῶν</ex>
-                    </expan>
-                    Ἰανουαρί-<lb n="7"/> ων. τὸν δὲ θεόν σε, φίλε,<lb n="8"/> μή μου σκύλῃς
-                    τὸν <supplied reason="omitted">β</supplied>όθ-<lb n="9"/> ρον, μή μοι δίξῃς φῶς.<lb n="10"/>
-                     ἂν δὲ θελήσῃς φῶς μοι<lb n="11"/> δῖξε, σοὶ τὸ φῶς<lb n="12"/> ὁ
-                    <expan>
-                        <abbr>θ</abbr>
-                        <ex>εὸ</ex>
-                        <abbr>ς</abbr>
-                    </expan>
-                    χόλιον δώση.<lb n="13"/> α𐆠ω ΙΧΘΥϹ <surplus>²⁶Ἰ(ησοῦς) Χ(ριστὸς) θ(εοῦ) υ(ἱὸς)
-                    σ(ωτήρ)</surplus>²⁶.
-                    <lb n="14"/>
+                    <lb n="1"/>ἐνθ<supplied reason="lost">ά</supplied>δε κῖτε Μαρκι<hi rend="ligature">λίν</hi>η
+                    <lb n="2"/><g type="staurogram">&#65952;</g> σεμνὴ <supplied reason="lost"><expan><abbr>κ</abbr><ex>αὶ</ex></expan></supplied> ἄμεμπτος βιώσα<supplied reason="omitted">σα</supplied>
+                    <lb n="3"/>ἰς τὸν κόσμον τοῦτον ἀπε<surplus>κ</surplus>
+                    <lb n="4" break="no"/>χώρι πρὸς <supplied reason="omitted">τὸν</supplied> <supplied reason="omitted">Κ</supplied>ύριον ἐτῶν
+                    <lb n="5"/>α<g type="staurogram">&#65952;</g>ω <num value="17">ιζ</num> <orig>Α<hi rend="supraline">Ι</hi>ΧΗΝ</orig> πράττουσα τῇ
+                    <lb n="6"/>πρὸ <num value="9">θ</num> <expan><abbr>καλ</abbr><ex>ανδῶν</ex></expan> Ἰανουαρί
+                    <lb n="7" break="no"/>ων τὸν δὲ θεόν σε φίλε
+                    <lb n="8"/>μή μου σκύλῃς τὸν <unclear>β</unclear>όθ
+                    <lb n="9" break="no"/>ρον μή μοι δίξῃς φῶς
+                    <lb n="10"/>ἂ<supplied reason="omitted">ν</supplied> δὲ θελήσῃς φῶς μοι
+                    <lb n="11"/>δῖξε σοὶ τὸ φῶς
+                    <lb n="12"/>ὁ <hi rend="ligature"><expan><abbr>θ</abbr><ex>εὸ</ex><abbr>ς</abbr></expan></hi> χόλιον δώση
+                    <lb n="13"/><g type="staurogram">&#65952;</g> <hi rend="supraline"><expan><abbr>ἰ</abbr><ex>ησοῦς</ex> <abbr>χ</abbr><ex>ριστὸς</ex> <abbr>θ</abbr><ex>εοῦ</ex> <abbr>ύ</abbr><ex>ἱὸς</ex> <abbr>ς</abbr><ex>ωτήρ</ex></expan></hi>
```

Comment:
again, hi rend supraline needs to be split up and put on each abbr individually I'm afraid